src/pages/guide/screens/Guide/__tests__/Guide.spec.tsx

```diff
@@ -35,11 +34,11 @@ describe('Guide', () => {
         render(<Guide />);
         expect(screen.getByText('Get started with P2P')).toBeInTheDocument();
     });
-    it('should navigate back to Buy/Sell page on click of return', async () => {
+    it('should navigate back to Buy/Sell page on click of return', () => {
         render(<Guide />);
 
         const backButton = screen.getByTestId('dt_page_return_btn');
-        await userEvent.click(backButton);
+        fireEvent.click(backButton);
```

Comment:
why did we change this?

Reply:
i have found that our test cases are slow when we try to run all of them mostly due to the userEvent calls, so i just wanted to replace this here since originally i was going to add another test here in this file. But in the future, if we want to speed the test cases, we need to revise where we should be using userEvent vs fireEvent